metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -399,6 +408,13 @@ def from_exported_bigquery_audit_metadata(
                 BigQueryTableRef.from_string_name(spec) for spec in raw_ref_tables
             ]
 
+        raw_ref_views = job["jobStats"]["queryStats"].get("referencedViews")
+        referenced_views = None
+        if raw_ref_views:
+            referenced_views = [
+                BigQueryTableRef.from_string_name(spec) for spec in raw_ref_views
+            ]
+
         try:
             statementType = job["jobConfiguration"]["query"]["statementType"]
```

Comment:
@treff7es / @anshbansal: Is this a bug? Should this instead be  `job["jobConfig"]["queryConfig"]["statementType"]`?

Reply:
based on google's documentation I think you are right, it should be just configuration -> https://cloud.google.com/bigquery/docs/reference/rest/v2/Job